storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/MySqlDriverHelper.java

```diff
@@ -36,10 +36,10 @@
 public class MySqlDriverHelper {
     public static final String MYSQL_JAR_FILE_PATTERN = "mysql-connector-java.*?.jar";
 
-    public static void downloadMySQLJarIfNeeded(StorageProviderConfiguration storageProperties,
-                                                String bootstrapDirPath,
-                                                String mysqlJarUrl,
-                                                Proxy proxy) throws Exception {
+    public static ClassLoader downloadMySQLJarIfNeeded(StorageProviderConfiguration storageProperties,
```

Comment:
May be we can call this loadMySQLJarIfNeeded() ?

Reply:
This is a public method, Is it ok to change the method name?

Reply:
Renamed to `maybeLoadMySQLJar`.